hackerrank/src/Algorithms/Implementation/TheTimeInWords.cs

```diff
@@ -0,0 +1,51 @@
+﻿namespace Fedotkin.Dotnet.HackerRank.Algorithms.Implementation.TheTimeInWords;
+
+public class Result
+{
+    private static string Word(int number)
+    {
+        switch (number)
+        {
+            case 1: return "one";
+            case 2: return "two";
+            case 3: return "three";
+            case 4: return "four";
+            case 5: return "five";
+            case 6: return "six";
+            case 7: return "seven";
+            case 8: return "eight";
+            case 9: return "nine";
+            case 10: return "ten";
+            case 11: return "eleven";
+            case 12: return "twelve";
+            case 13: return "thirteen";
+            case 14: return "fourteen";
+            case 16: return "sixteen";
+            case 17: return "seventeen";
+            case 18: return "eighteen";
+            case 19: return "nineteen";
+            case 20: return "twenty";
+            default: return string.Concat(Word(20), " ", Word(number - 20)); ;
+        }
+    }
+
+    public static string TimeInWords(int h, int m)
+    {
+        if (m == 0)
+            return string.Concat(Word(h), " o' clock");
+        else if (m == 1)
+            return string.Concat(Word(m), " minute past ", Word(h));
+        else if (m == 15)
+            return string.Concat("quarter past ", Word(h));
+        else if (1 < m & m < 30)
```

Comment:
```suggestion
        else if (1 < m && m < 30)
```
Розніцу паміж `&` ды `&&` разумееш?

Reply:
Праверка другой умовы не праводзіцца, калі вынік першай умовы - false